app/screens/global_drafts/index.tsx

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo} from 'react';
+import {useIntl} from 'react-intl';
+import {Keyboard, StyleSheet, View} from 'react-native';
+import {SafeAreaView, type Edge} from 'react-native-safe-area-context';
+
+import NavigationHeader from '@components/navigation_header';
+import OtherMentionsBadge from '@components/other_mentions_badge';
+import RoundedHeaderContext from '@components/rounded_header_context';
+import {Screens} from '@constants';
+import {useIsTablet} from '@hooks/device';
+import {useDefaultHeaderHeight} from '@hooks/header';
+import {useTeamSwitch} from '@hooks/team_switch';
+
+import {popTopScreen} from '../navigation';
+
+import GlobalDraftsList from './components/global_drafts_list';
+
+import type {AvailableScreens} from '@typings/screens/navigation';
+
+const edges: Edge[] = ['left', 'right'];
```

Comment:
Why not include bottom? Is it added elsewhere? I see top in the styles but not bottom. Or is it that we don't want it at all?

Reply:
This is what the design suggests https://www.figma.com/design/VbEx15Tpvi9mXhs9IEjJtq/MM-45558-Drafts-%26-Schedule-Messages?node-id=4013-70751&m=dev, also we have not used it in the Threads View.